cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/query/IndexManager.java

```diff
@@ -385,23 +361,27 @@ protected Datastore getDatastore() {
     }
 
     protected ExecutorService getQueue() {
-        return queue;
+        return null;
```

Comment:
I think we'd be better off deleting this rather than returning `null` and equally for `getDatabase`. I think the chances of anyone calling/using them in a subclass are remote.